influxdb_iox/tests/end_to_end_cases/cli.rs

```diff
@@ -10,13 +10,16 @@ use test_helpers_end_to_end::{maybe_skip_integration, MiniCluster, Step, StepTes
 
 #[tokio::test]
 async fn default_mode_is_run_all_in_one() {
+    let tmpdir = tempdir().unwrap();
+
     Command::cargo_bin("influxdb_iox")
         .unwrap()
         .args(&["-v"])
         // Do not attempt to connect to the real DB (object store, etc) if the
         // prod DSN is set - all other tests use TEST_INFLUXDB_IOX_CATALOG_DSN
         // but this one will use the real env if not cleared.
         .env_clear()
+        .env("HOME", tmpdir.path())
```

Comment:
Without this, we have errors about writing read-only root filesystem on macos.

Reply:
🤔 
```suggestion
        // Without this, we have errors about writing read-only root filesystem on macos.
        .env("HOME", tmpdir.path())
```

For what it is worth I have not had these issues, so perhaps it is related to something in your environment. In any event, this looks like a reasonable change to me

Reply:
I wonder if you have a `.env` file or something that affects the file location, or have the root directory `/` mounted as writable? It was 100% repeatably failing for me.